src/main/webapp/app/shared/table/OncoKBTable.tsx

```diff
@@ -1,96 +1,76 @@
-import React, { useMemo } from 'react';
-import { Table } from 'reactstrap';
-import { Column, useAbsoluteLayout, useBlockLayout, useFlexLayout, useResizeColumns, useTable } from 'react-table';
-import './OncoKBTable.scss';
-import LoadingIndicator from 'app/oncokb-commons/components/loadingIndicator/LoadingIndicator';
+import React, { useMemo, useState } from 'react';
```

Comment:
This component is mostly copied over from oncokb-public. However, I did convert from a class component to a functional component because the mobx decorators don't work with newer mobx version.